workos/__about__.py

```diff
@@ -12,7 +12,7 @@
 
 __package_url__ = "https://github.com/workos-inc/workos-python"
 
-__version__ = "1.14.0"
+__version__ = "1.15.0"
```

Comment:
We typically bump the versions in a dedicated branch after feature changes have been merged in.